extensions/ql-vscode/src/extension.ts

```diff
@@ -349,6 +349,7 @@ async function activateWithInstalledDistribution(
     getContextStoragePath(ctx),
     ctx.extensionPath
   );
+  databaseUI.init();
```

Comment:
Can this call be part of the database UI constructor?

Reply:
No. The tests will fail since it registers commands. And if we create an instance more than once and re-register commands, there is an error. 

I guess I could alternatively add an optional flag to signal the avoidance of command registration. But, that seems uglier to me.